tox.ini

```diff
@@ -57,3 +58,8 @@ basepython=python3
 deps=isort==5.9.1
 changedir={toxinidir}/tests
 commands=isort --thirdparty aiohttp,connexion --check-only --diff .
+
+[testenv:mypy]
+deps=mypy==0.910
+ignore_outcome=true
+commands=mypy --exclude '/(__init__|api|api.pets|app|hello|orm|resty)\.py$' --install-types --non-interactive .
```

Comment:
Looking at the commits you made, I believe using `--exclude 'examples/'` should do the trick?
Or could we simply do `mypy connexion` (not sure about a potential `-m` or `-p` flag)?